content/overview/get-started/gs-setup-data-hub-users.md

```diff
@@ -8,12 +8,12 @@ Add the groups that you created in AVEVA Connect to AVEVA Data Hub, and then ass
 
 **Step 3A: Log In to AVEVA Data Hub**
 
-From AVEVA Connect, log into AVEVA Data Hub. For instructions, see <xref:gs-overview>.
+From AVEVA Connect, log into AVEVA Data Hub. For instructions, see <xref:loginADH>.
 
 **Step 3B: Add groups and assign roles**
 
 Add the groups that you created in AVEVA Connect to AVEVA Data Hub. Then assign one or more roles to the groups. For instructions, see <xref:map-group-adh>.
 
-**Step 3C: Manage namespace permissions**
+**Step 3C: Manage permissions**
```

Comment:
It might be a good idea to divide step 3c into two separate steps: "Manage permissions for namespaces" and "Manage permissions for resources".